services/state/token/account.proto

```diff
@@ -0,0 +1,176 @@
+ */
+
+message Account {
+    /**
+     * The unique entity number of the account.
+     */
+    int64 account_number = 1;
+    /**
+     * The public key bytes to be used as the account's alias, if any.
+     */
+    bytes alias = 2;
+    /**
+     * (Optional) The key to be used to sign transactions from the account, if any.
+     */
+    Key key = 3;
+    /**
+     * The expiration time of the account, in seconds since the epoch.
+     */
+    int64 expiry = 4;
+    /**
+     * The balance of the account, in tiny-bars.
+     */
+    int64 tinybar_balance = 5;
+    /**
+     * An optional description of the account with UTF-8 encoding up to 100 bytes.
+     */
+    string memo = 6;
+    /**
+     * A boolean marking if the account has been deleted.
+     */
+    bool deleted = 7;
+    /**
+     * A boolean marking if the account is a smart contract.
+     */
+    bool smart_contract = 8;
+    /**
+     * A boolean marking if the account requires a receiver signature.
+     */
+    bool receiver_sig_required = 9;
+    /**
+     * The number of NFTs owned by the account.
+     */
+    int64 number_owned_nfts = 10;
+    /**
+     * The maximum number of tokens that can be auto-associated with the account.
+     */
+    int32 max_auto_associations = 11;
+    /**
+     * The number of used auto-association slots.
+     */
+    int32 used_auto_associations = 12;
+    /**
+     * The number of tokens associated with the account.
+     */
+    int32 number_associations = 13;
+    /**
+     * The number of tokens with a positive balance associated with the account.
+     */
+    int32 number_positive_balances = 14;
+    /**
+     * The nonce of the account, used for Ethereum interoperability.
+     */
+    int64 ethereum_nonce = 15;
+    /**
+     * The amount of hbars staked to the account.
+     */
+    int64 staked_to_me = 16;
+    /**
+     * If this account stakes to another account, its value will be -1. It will
+     * be set to the time when the account starts staking to a node.
+     */
+    int64 stake_period_start = 17;
+    /**
+     * The node number or the account number this account is staked to.
+     * It is negative if staking to a node and positive if staking to an account and 0 if not staking to anyone.
+     * When staking to a node, it is stored as -node-1 to differentiate node 0.
+     */
+    int64 staked_number = 18;
+    /**
+     * A boolean marking if the account declines rewards.
+     */
+    bool decline_reward = 19;
+    /**
+     * The amount of hbars staked to the account at the start of the last rewarded period.
+     */
+    int64 stake_at_start_of_last_rewarded_period = 20;
+    /**
+     * (Optional) The number of an auto-renew account, in the same shard and realm as the account, that
+     * has signed a transaction allowing the network to use its balance to automatically extend the account's
+     * expiration time when it passes.
+     */
+    int64 auto_renew_account_number = 21;
```

Comment:
If we end up with shard and realm encoded in these files, we'll need it here as well. Otherwise we can leave it off.